build-system/tasks/check-links.js

```diff
@@ -135,7 +135,12 @@ function filterWhitelistedLinks(markdown) {
   filteredMarkdown = filteredMarkdown.replace(/<code>(.*?)<\/code>/g, '');
 
   // Dailymotion link that is marked dead in Travis CI
-  filteredMarkdown = filteredMarkdown.replace(/https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
+  filteredMarkdown = filteredMarkdown.replace(
+    /https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
+
+  // Font providers CDNs that return 404 on the root
+  filteredMarkdown = filteredMarkdown.replace(/https:\/\/fast.fonts.net/g, '');
+  filteredMarkdown = filteredMarkdown.replace(/https:\/\/fonts.googleapis.com/g, '');
```

Comment:
No problem, I wasn't sure what is the better solution :)

Thanks for comments!